support-frontend/assets/pages/paper-subscription-checkout/components/addDigiSubCta.jsx

```diff
@@ -0,0 +1,110 @@
+// @flow
+
+import React from 'react';
+import { css } from '@emotion/core';
+import { headline } from '@guardian/src-foundations/typography';
+import { space } from '@guardian/src-foundations';
+import { neutral } from '@guardian/src-foundations/palette';
+import { from } from '@guardian/src-foundations/mq';
+import { Accordion, AccordionRow } from '@guardian/src-accordion';
+import { Checkbox } from '@guardian/src-checkbox';
+import GridImage from 'components/gridImage/gridImage';
+import { ListWithSubText } from 'components/list/list';
+
+
+type PropTypes = {|
+  digiSubPrice: string;
+  addDigitalSubscription: (event: SyntheticInputEvent<HTMLInputElement>) => void
+|}
+
+const rowOverrides = css`
+  border: none;
+  & > div {
+    max-height: 1000px;
+  }
+
+  & button strong {
+    ${headline.xsmall({ fontWeight: 'bold' })}
+  }
+`;
+
+const ctaContainer = css`
+  background-color: ${neutral[97]};
+  padding: 0 ${space[6]}px;
+  border-top: 1px solid ${neutral[86]};
+  border-bottom: 1px solid ${neutral[86]};
+`;
+
+const lightBorder = css`
+  border-bottom: 1px solid ${neutral[86]};
+`;
+
+const imageContainer = css`
+  display: flex;
+  justify-content: center;
+`;
+
+const heading = css`
+  ${headline.xsmall({ fontWeight: 'bold' })}
+  margin-bottom: ${space[3]}px;
+`;
+
+const content = css`
+  padding: ${space[6]}px 0;
+`;
+
+const list = css`
+  margin-bottom: 0;
+  ${from.desktop} {
+    margin-bottom: 0;
+  }
+`;
+
+const listCopy = [
+  {
+    content: 'The Guardian Editions app',
+    subText: `Each edition available to read by 6am (GMT), 7 days a week for
+    IOS and Android mobile and tablet. Download and read whenever it suits you.`,
+  },
+  {
+    content: 'Premium access to The Guardian Live app',
+    subText: 'Live feed, Discover and daily crosswords. Download the news whenever it suits you.',
+  },
+  {
+    content: 'Ad-free',
+    subText: 'Enjoy our journalism uninterrupted, without adverts',
+  },
+];
+
+function AddDigiSubCta({ addDigitalSubscription, digiSubPrice }: PropTypes) {
+  return (
+    <Accordion cssOverrides={ctaContainer} hideToggleLabel>
+      <AccordionRow cssOverrides={rowOverrides} label={`Would you like to add a Digital subscription for ${digiSubPrice.replace(/\/(.*)/, '')}?`}>
```

Comment:
When using regex, it's probably nice to give it a semantic name so people can easily see what it's doing.